spec/datetimeformat.html

```diff
@@ -247,37 +251,99 @@
         1. Perform ! CreateDataPropertyOrThrow(_nf2Options_, `"useGrouping"`, *false*).
         1. Let _nf2_ be ? Construct(%NumberFormat%, « _nfLocale_, _nf2Options_ »).
```

Comment:
Once again, I believe this construct and the other 4 lines above won't return any abrupt completion.